packages/react/src/components/Inbox.tsx

```diff
@@ -0,0 +1,66 @@
+import type { InboxNotification } from '@novu/js';
+import type { BaseNovuUIOptions } from '@novu/js/ui';
+import React from 'react';
+import { useRenderer } from '../context/RenderContext';
+
+import { Mounter } from './Mounter';
+import { Renderer } from './Renderer';
+
+type InboxDefaultProps = {
+  renderNotification?: (notification: InboxNotification) => React.ReactNode;
+  renderBell?: ({ unreadCount }: { unreadCount: number }) => React.ReactNode;
+};
+
+const InboxDefault = (props: InboxDefaultProps) => {
+  const { renderNotification, renderBell, ...rest } = props;
+  const { novuUI, mountElement } = useRenderer();
+
+  const mount = React.useCallback(
+    (element: HTMLElement) => {
+      return novuUI.mountComponent({
+        name: 'Inbox',
+        props: {
+          ...rest,
+          mountNotification: renderNotification
```

Comment:
Should this prop be renderNotification?
```suggestion
          mountNotification: renderNotification
```